### PR TITLE
Bump canonicalwebteam.search

### PR DESCRIPTION
**Pull request by petesfrench**
## Done

- Updates to the newest version of search module

## QA

- Check out the branch locally and update the request limit to a low number (1 or 2) and check the rate gets limited.
- Check the search works

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-12317

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -32,3 +33,4 @@ bleach==6.0.0
 numpy==1.24.4
 python-slugify==8.0.1
 djlint==1.34.1
+Flask-Limiter==1.4
```
Comment (carkod) [inline requirements.txt:36]: This should't be needed if canonicalwebteam.search is included

File: webapp/app.py [hunk 2]
```diff
@@ -408,7 +408,7 @@
         session=session,
         template_path="search.html",
         search_engine_id=search_engine_id,
-        request_limit="2000/day",
+        request_limit="2/day",
```
Comment (carkod) [inline webapp/app.py:411]: QAed, don't forget to restore this to `2000`

**Review by samhotep — APPROVED** [review]
LGTM!
@petesfrench any idea why playwright is timing out?

**Review by samhotep — COMMENTED** [review]
Also, any reason why we're not using version 2.0.0?

**Comment by petesfrench** [comment]
@samhotep Updated now. I had bumped the version number incorrectly, so 1.4.0 and 2.0.0 are the same, but better to have it on 2.0.0 as you say.

**Review by samhotep — APPROVED** [review]

**Pull request merged**